core/kubernetes/dev/dev-urls.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+res="$(kubectl get service openhim-mongo-dev-service)"
```

Comment:
We should name this variable a bit more descriptive in terms of what response it is.